addons/hearing/stringtable.xml

```diff
@@ -300,27 +302,31 @@
         </Key>
         <Key ID="STR_ACE_Hearing_earplugsVolume_DisplayName">
             <English>Earplugs Volume</English>
+            <German>Lautstärke bei Ohrenstöpsel </German>
             <Japanese>耳栓時の音量</Japanese>
             <Chinesesimp>耳塞时音量</Chinesesimp>
             <Chinese>耳塞時音量</Chinese>
             <Italian>Volume tappi per le orecchie</Italian>
         </Key>
         <Key ID="STR_ACE_Hearing_earplugsVolume_Description">
             <English>Volume when using earplugs.</English>
+            <German>Lautstärke bei angelegten Ohrenstöpsel </German>
             <Japanese>耳栓使用時の音量を決定します。</Japanese>
             <Chinesesimp>决定带上耳塞时的音量</Chinesesimp>
             <Chinese>使用耳塞時音量</Chinese>
             <Italian>Volume audio quandi si indossano i tappi per le orecchie.</Italian>
         </Key>
         <Key ID="STR_ACE_Hearing_unconsciousnessVolume_DisplayName">
             <English>Unconscious Volume</English>
+            <German>Lautstärke bei Bewusstlosigkeit </German>
             <Japanese>気絶時の音量</Japanese>
             <Chinesesimp>无意识时音量</Chinesesimp>
             <Chinese>昏迷時音量</Chinese>
             <Italian>Volume quando incoscente</Italian>
         </Key>
         <Key ID="STR_ACE_Hearing_unconsciousnessVolume_Description">
             <English>Volume when unconscious.</English>
+            <German>Umgebungslautstärke bei bewusstlosem Spieler </German>
```

Comment:
"Reduziert die Umgebungslautstärke wenn ein Spieler bewusstlos ist."
Leerzeichen zu viel

Reply:
The play himself. Not just some other player.
so "wenn der Spieler"